File: ComputerClient/src/app/models/computer.ts
export class Computer {
    constructor(
        private _id: number, 
        private _model:string, 
        public ram:number,
        public price:number){}

    
    get id() : number {
        return this._id;
    }

    get model() : string{
        return this._model;
    }
    
    // get ram() : number {
    //     return this._ram;
    // }

    // get price() : number {
    //     return this._price;
    // }
    
    // public set set_ram(v : number) {
    //     this._ram = v;
    // }

    // public set set_price(v : number) {
    //     this._price = v;
    // }
}

File: Unorganized/ComputerServer/src/main/java/com/example/ComputerServer/beans/Computer.java
package com.example.ComputerServer.beans;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.Table;

@Entity
@Table(name="computers")
public class Computer {
	
	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	private long id;
	private String model;
	private int ram;
	private double price;
	
	public Computer() {}

	public Computer(String model, int ram, double price) {
		super();
		this.model = model;
		this.ram = ram;
		this.price = price;
	}

	public int getRam() {
		return ram;
	}

	public void setRam(int ram) {
		this.ram = ram;
	}

	public double getPrice() {
		return price;
	}

	public void setPrice(double price) {
		this.price = price;
	}

	public long getId() {
		return id;
	}

	public String getModel() {
		return model;
	}

	@Override
	public String toString() {
		return "Computer [id=" + id + ", model=" + model + ", ram=" + ram + ", price=" + price + "]";
	}
	
	
}
